src/java.desktop/windows/native/libawt/windows/awt_PrintJob.cpp

```diff
@@ -209,7 +211,6 @@ static const double POINTS_TO_HIMETRIC = (2540.0 / 72.0);
  */
 static const double POINTS_TO_LOMETRIC = (254.0 / 72.0);
 
-jfieldID AwtPrintDialog::pageID;
 
```

Comment:
I recommend deleting one more line, since otherwise you will now have three consecutive blank lines.